Squared ISWAP fails trace distance bound test

```
In [1]: import cirq

In [2]: cirq.testing.consistent_protocols.assert_has_consistent_trace_distance_bound(cirq.ISWAP**2)
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-2-9d7942dfa2d6> in <module>
----> 1 cirq.testing.consistent_protocols.assert_has_consistent_trace_distance_bound(cirq.ISWAP**2)

~/src/Cirq/cirq/testing/consistent_protocols.py in assert_has_consistent_trace_distance_bound(val)
    109 
    110         assert val_from_trace >= val_from_unitary or np.isclose(
--> 111             val_from_trace, val_from_unitary)
    112 
    113 

AssertionError: 

In [3]: cirq.trace_distance_bound(cirq.ISWAP**2)
Out[3]: 1.2246467991473532e-16
```

I guess we really care about this near identity, but still the protocol explicitly says "Underestimates are not permitted.", so we should fix it in case it's relied on in future.

Other exponents in [1.0029..2.9971] also fail the check.

--- a/cirq/ops/common_gates.py
+++ b/cirq/ops/common_gates.py
@@ -1287,11 +1287,6 @@ class ISwapPowGate(eigen_gate.EigenGate,
                              [0, 0, 0, 0]])),
         ]
 
-    def _trace_distance_bound_(self) -> Optional[float]:
-        if self._is_parameterized_():
-            return None
-        return abs(np.sin(self._exponent * 0.5 * np.pi))
-
     def _decompose_(self, qubits):
         a, b = qubits
 

--- a/cirq/ops/common_gates_test.py
+++ b/cirq/ops/common_gates_test.py
@@ -737,9 +737,11 @@ def test_iswap_unitary():
         np.array([[1, 0, 0, 0], [0, 0, 1j, 0], [0, 1j, 0, 0], [0, 0, 0, 1]]),
         atol=1e-8)
 
+
 def test_iswap_trace_distance():
     for exp in np.linspace(0, 4, 20):
         cirq.testing.assert_has_consistent_trace_distance_bound(cirq.ISWAP**exp)
+
 
 def test_iswap_decompose_diagram():
     a = cirq.NamedQubit('a')

--- a/cirq/ops/common_gates_test.py
+++ b/cirq/ops/common_gates_test.py
@@ -738,7 +738,7 @@ def test_iswap_unitary():
         atol=1e-8)
 
 def test_iswap_trace_distance():
-    for exp in np.linspace(0,4,20):
+    for exp in np.linspace(0, 4, 20):
         cirq.testing.assert_has_consistent_trace_distance_bound(cirq.ISWAP**exp)
 
 def test_iswap_decompose_diagram():

--- a/cirq/ops/common_gates_test.py
+++ b/cirq/ops/common_gates_test.py
@@ -737,6 +737,9 @@ def test_iswap_unitary():
         np.array([[1, 0, 0, 0], [0, 0, 1j, 0], [0, 1j, 0, 0], [0, 0, 0, 1]]),
         atol=1e-8)
 
+def test_iswap_trace_distance():
+    for exp in np.linspace(0,4,20):
+        cirq.testing.assert_has_consistent_trace_distance_bound(cirq.ISWAP**exp)
 
 def test_iswap_decompose_diagram():
     a = cirq.NamedQubit('a')

